week-4/investigate/find.js

```diff
@@ -20,6 +20,10 @@ console.log(find("code your future", "z"));
 // Pay particular attention to the following:
 
 // a) How the index variable updates during the call to find
+// it is incremented at each loop pass
```

Comment:
Correct.

I see where you got the while loop habit from - I wouldn't copy this example though - it's not the best. You need to understand the diffrent loop types and use them in the right places to make your code as easy to read and bug free as possible